### ISSUE TITLE
Not monitoring on default endpoint

### ISSUE THREAD

**Issue by RipinAggarwal** [microsoft/vscode-azure-iot-toolkit#491]
Hi Team,

Throwing error when trying to monitor on default-endpoint of a device.
Error:
[IoTHubMonitor] Start monitoring message arrived in built-in endpoint for device [myTempDevice] ...
[IoTHubMonitor] TypeError: Either provide "eventHubName" or the "connectionString": "HostName=iothub1451.azure-devices.net;SharedAccessKeyName=iothubowner;SharedAccessKey=XXXX", must contain "EntityPath=<your-event-hub-name>".

Regards,
Ripin Aggarwal

**Comment by rubberduck203** [comment]
I'm also seeing this.
Looks like some of OP's message got cut off due to the angle brackets.
Full error message below.

```
[IoTHubMonitor] TypeError: Either provide "eventHubName" or the "connectionString": "HostName=ducks-iot-hub.azure-devices.net;SharedAccessKeyName=iothubowner;SharedAccessKey=XXX", must contain "EntityPath=<your-event-hub-name>".
```

**Comment by rubberduck203** [comment]
This d2c read example is also currently broken.
https://github.com/Azure-Samples/azure-iot-samples-csharp/blob/master/iot-hub/Quickstarts/read-d2c-messages/ReadDeviceToCloudMessages.cs#L43

I was able to get it working by switching to the [`EventHubConsumerClient(string,string)` overload](https://docs.microsoft.com/en-us/dotnet/api/azure.messaging.eventhubs.consumer.eventhubconsumerclient.-ctor?view=azure-dotnet#Azure_Messaging_EventHubs_Consumer_EventHubConsumerClient__ctor_System_String_System_String_) and including the `EntityPath` directly in the connection string.

**Comment by dattosma** [comment]
Same issue after updating the Azure IoT Hub extension to 2.16.3 yesterday - any idea how to revert back to a previous version ?

**Comment by kushaljh** [comment]
Encountering the same issue since yesterday afternoon after updating to 2.16.3

**Comment by mrarmyant** [comment]
Can confirm, as bugs are being reported in a package that installs this.  https://github.com/microsoft/vscode-azure-iot-tools/issues/30

**Comment by formulahendry** [comment]
This is a regression in v2.16.3: https://github.com/microsoft/vscode-azure-iot-toolkit/pull/487 Due to some Azure npm packages are changed,

**Comment by montgomp** [comment]
Resolved in release v2.16.4
